services/121-service/src/payments/services/payments-execution.service.ts

```diff
@@ -80,17 +70,15 @@ export class PaymentsExecutionService {
         },
       );
 
-      void this.createTransactionJobs({
+      await this.createTransactionJobs({
         programId,
         transactionIds: transactionsOfIncludedRegistrations.map((t) => t.id),
         userId,
         isRetry: false,
       });
     } finally {
```

Comment:
I added the await here:
- If we do not await here there is a significant time window where the payment is not lock but there are no transaction jobs in the queue
- another option is to have a .catch, .than at createTransactionJobs

but since we want to increase performance anyway I am now puzzeling with that

Reply:
- Sure good point, but how long does startPayment now take for e.g. 100k transactions?
- Also the performance you're working on was on createPayment right? so unrelated?